sci-geosciences/gmt/gmt-5.4.4.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit bash-completion-r1 cmake-utils
+
+DESCRIPTION="Powerful map generator"
+HOMEPAGE="https://gmt.soest.hawaii.edu/"
+SRC_URI="mirror://gmt/${P}-src.tar.xz"
+
+LICENSE="GPL-2 gmttria? ( Artistic )"
```

Comment:
Here the license has been changed probably.

Reply:
GPL-3+. See above about the license for the triangulation code.